torch/_inductor/compile_fx.py

```diff
@@ -679,6 +679,8 @@ def fw_compiler_freezing(
     # partition_fn won't be called
     joint_graph_passes(aot_autograd_model)
 
+    convert_conv_weights_to_channels_last(aot_autograd_model, aot_example_inputs)
+
```

Comment:
I have a question about it, for convolution, if input or weight has one channels_last, the output format will be channels_last, if we do a format change there, do we need to update the  ```node.meta``` info(there have such infos: {"val": FakeTensor, "tensor_meta": TensorMetadata})?

Reply:
yea, we need change both layout for compile time input (in `Node.meta['val']`) and layout for runtime input to make them consistent.

BTW, this PR is WIP. I have some related comments in another file but have not implement that yet. I'll remove 'WIP' tag in the title when the PR is ready.